SPECS/fluent-bit/fluent-bit.spec

```diff
@@ -75,6 +75,9 @@ Development files for %{name}
 %{_libdir}/fluent-bit/*.so
 
 %changelog
+* Tue May 28 2024 Neha Agarwal <nehaagarwal@microsoft.com> - 3.0.6-1
```

Comment:
Changes look good, @neha170. Could I ask you to do one more thing, since you're here, though? It looks like at some point we missed migrating an update adding tests to `fluent-bit` from 2.0. Could you add the test-related changes from #6991? It's not a blocker for this PR, but definitely something we'd like to have.